index.html

```diff
@@ -4,7 +4,7 @@
   <meta charset='utf-8'>
   <meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no'>
   <meta http-equiv='x-ua-compatible' content='ie=edge'>
-  <title>Pokémon GO - LocationSpoof</title>
+  <title>Pokémon GO - LocationSpoof v1.1.0</title>
```

Comment:
@tiagocardosoweb - You should really pull the version from the package.json so this doesn't need to be updated manually every time a new version is released.